### PR TITLE
[11.0] [MIG] sale_triple_discount

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Sale Triple Discount
====================

- This module allows having three successive discounts on every sale order
line.

Migration of module Sale Triple Discount v10 to v11.

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
account_invoice_triple_discount is missing it says. In your PR, add a line for account-invoicing in oca_dependencies.txt

File: sale_triple_discount/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    )
+    discount3 = fields.Float(
+        'Disc. 3 (%)',
+        digits=dp.get_precision('Discount'),
+        default=0.0,
+    )
+
+    _sql_constraints = [
+        ('discount2_limit', 'CHECK (discount2 <= 100.0)',
+         'Discount 2 must be lower than 100%.'),
```
Comment (JayVora-SerpentCS) [inline sale_triple_discount/models/sale_order_line.py:45]: Lets use _(''Message")

**Review by JayVora-SerpentCS — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Please don't open a new PR if the other one has the same problem. Let's work on the other PR. I have already pointed out which can be the problem in https://github.com/OCA/sale-workflow/pull/627#issuecomment-403040749

**Comment by JayVora-SerpentCS** [comment]
Agreed, @murtuzasaleh missed it. #627 has been reviewed. Thanks.

**Pull request closed without merging**